client/luigi-client.d.ts

```diff
@@ -612,10 +612,19 @@ export type getEventData = () => Context;
  */
 export function getContext(): Context;
 export type getContext = () => Context;
-
+/**
+ * Set search query parameters to Luigi core. It will be added to url.
```

Comment:
@stanleychh This comment from @alexandra-simeonova needs to be resolved. 

One more note: **addNodeParams** comment should be 'Set ~search query~ **node** parameters' in the function comment and docu part.